source/features/warn-pr-from-master.tsx

```diff
@@ -27,14 +28,24 @@ async function init(): Promise<false | void> {
 	);
 }
 
+function init(signal: AbortSignal): void {
+	observe('.is-cross-repo', warnPR, {signal});
+}
+
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.isCompare,
 	],
 	exclude: [
 		pageDetect.isBlank,
 	],
-	awaitDomReady: true,
-	deduplicate: 'has-rgh',
 	init,
 });
+
+/*
+
+Test URLs
+
+https://github.com/refined-github/refined-github/compare/sandbox/keep-branch...yakov116:upstream
```

Comment:
The `upstream` name is confusing, let's use this one

```suggestion
https://github.com/refined-github/refined-github/compare/main...fregante:refined-github:main
```